articles/types/jsdoc.md

```diff
@@ -1,3 +1,179 @@
 # JsDoc support
 
-TODO
+Programs written in JavaScript and TypeScript can be annotated with (JsDoc comments)[http://usejsdoc.org/index.html]
+to give type information and quick-info descriptions of identifiers.
+
+In the following example, the text between the opening `/**` and closing `/*` are considered JsDoc comments.
```

Comment:
(1) "the text ... is considered" (2) typo for comment close: `*/`

Reply:
changed

Reply:
Still has mismatch between "the text ... are"